federation-js/src/composition/validate/postComposition/__tests__/providesNotOnEntity.test.ts

```diff
@@ -127,6 +129,12 @@ describe('providesNotOnEntity', () => {
       Array [
         Object {
           "code": "PROVIDES_NOT_ON_ENTITY",
+          "locations": Array [
```

Comment:
Should this location point to the return type of the field? (`LineItem`)
Alternatively, maybe on the `@provides` directive itself. Thoughts?

Reply:
Yeah I think the provides makes the most sense here. 👍